core-web/libs/ui/src/lib/components/dot-workflow-actions/dot-workflow-actions.component.html

```diff
@@ -1,20 +1,21 @@
 @for(action of groupedActions(); track $index; let first = $first) { @if(action.length > 1) {
 <p-splitButton
     [disabled]="loading"
-    [styleClass]="!first ? 'p-button-outlined' : ''"
+    [styleClass]="styleClass(!first)"
```

Comment:
Maybe this can be a pipe? Calling functions here scares me

Reply:
I prefer `[styleClass]="!first ? 'p-button-outlined' : ''"` idk

Reply:
I'll do it inline

Reply:
Done!